autospec/autospec.py

```diff
@@ -320,6 +321,10 @@ def package(args, url, name, archives, workingdir, infile_dict):
                                                tarball.name,
                                                tarball.version,
                                                tarball.release)
+        if filemanager.is_avx_candidate(mock_chroot):
+            print("Autospec found files candidates to AVX technology !")
```

Comment:
This is poor grammar, maybe "Autospec found file candidates for AVX optimizations"